.github/workflows/vast.yaml

```diff
@@ -513,3 +522,94 @@ jobs:
+      DESTDIR: '${{ github.workspace }}'
+    name: ${{ matrix.plugin.name }} Plugin
+    steps:
+      - name: Install Dependencies
+        run: |
+          apt-get update
+          apt-get -y install git libsnappy-dev libbrotli-dev zlib1g-dev wget liblz4-dev libzstd-dev ninja-build libpcap-dev libssl-dev libbenchmark-dev tcpdump lsb-release libflatbuffers-dev flatbuffers-compiler-dev libyaml-cpp-dev libsimdjson-dev gnupg2 gcc-8 g++-8 python3 python3-pip python3-venv jq apt-transport-https ca-certificates curl gnupg-agent software-properties-common
+          # Need to specify backports explicitly, since spdlog and fmt also have
+          # buster packages.
+          apt-get -y -t buster-backports install libspdlog-dev libfmt-dev
+          # Apache Arrow (c.f. https://arrow.apache.org/install/)
+          wget https://apache.jfrog.io/artifactory/arrow/$(lsb_release --id --short | tr 'A-Z' 'a-z')/apache-arrow-archive-keyring-latest-$(lsb_release --codename --short).deb
+          apt-get -y install ./apache-arrow-archive-keyring-latest-$(lsb_release --codename --short).deb
+          sed -i'' -e 's,https://apache.bintray.com/,https://apache.jfrog.io/artifactory/,g' /etc/apt/sources.list.d/apache-arrow.sources
+          apt-get update
+          apt-get -y install libarrow-dev
+          # Install CMake from pip
+          python3 -m pip install --upgrade pip
+          python3 -m pip install --upgrade cmake
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Fetch Submodules
+        run: |
+          git submodule update --init --recursive '${{ matrix.plugin.path }}'
+      - name: Determine VAST Package Name
+        id: configure
+        run: |
+          PACKAGE_NAME="$(echo "vast-$(git describe)-$(uname -s)-release-gcc" | awk '{ print tolower($0) }')"
+          echo "PACKAGE_NAME=$PACKAGE_NAME" >> $GITHUB_ENV
+      - name: Download VAST
+        uses: actions/download-artifact@v2
+        with:
+          name: '${{ env.PACKAGE_NAME }}.tar.gz'
+      - name: Install VAST
+        run: |
+          mkdir ${INSTALL_DIR}
+          tar -C ${INSTALL_DIR} -xzvf "${PACKAGE_NAME}.tar.gz" --strip-components 1
+          echo "${INSTALL_DIR}/bin" >> $GITHUB_PATH
+      - name: Configure Build
+        env:
+          VAST_DIR: '${{ env.INSTALL_DIR }}'
+        run: |
+          python3 --version
+          python3 -m pip --version
+          cmake --version
+          cmake -S '${{ matrix.plugin.path }}' -B "$BUILD_DIR"
+      - name: Build
+        run: |
+          cmake --build "$BUILD_DIR" --target all --parallel
+      - name: Run Unit Tests
+        run: |
+          cmake --build "$BUILD_DIR" --target test
+      - name: Run Integration Tests
+        id: integration_tests
+        run: |
+          echo "::set-output name=running=true"
+          cmake --build "$BUILD_DIR" --target integration
+          echo "::set-output name=running=false"
```

Comment:
What do we need the `running` output for?

Reply:
Leftover from a previous test, will remove.